### ISSUE TITLE
Add hand position to discharge alert?

### ISSUE THREAD

**Issue by terracoda** [phetsims/john-travoltage#236]
In John Travoltage, when hand moving fast I find it hard to remember at which position, the discharge occurs. 

The change just adds a phrase at the end:
- Discharge occurred. Electrons on body decreased from [start#] to [end#].
- Discharge occurred. Electrons on body decreased from [start#] to [end#] **with hand at [position #], [region name].**

I think it makes important information (hand position) more readily available on discharge, allowing the user to quickly explore where discharges occur with different amounts of charge on John's body.

**Examples with rough sample data (not real numbers)**
- Discharge occurred. Electrons on body decreased from 10 to 0 **with hand at position 0, at doorknob.**
- Discharge occurred. Electrons on body decreased from 25 to 0 **with hand at position 2, very close to doorknob.**
- Discharge occurred. Electrons on body decreased from 87 to 15 **with hand at position -7, not so close to doorknob.**

In the last example, the user moves during the discharge. In this case, the position of the arm on the slider and in the summary would be different from when the discharge actually happened, making the discharge info more difficult to directly compare with charge level. Adding position information directly in the alert seems quite helpful to me.

@jhung do you have any objections to adding the position and region name at end of the discharge alert?

@jessegreenberg, @jobara, @emily-phet, any thoughts?

**Comment by jhung** [comment]
@terracoda I agree that this would be helpful especially if the hand is moving very fast. +1 from me!

**Comment by terracoda** [comment]
@jhung, awesome! Glad you like the idea.

@jessegreenberg, let me know if you need anything on this. I know it is short notice given our timeline.

**Comment by jessegreenberg** [comment]
Sounds good @terracoda, here is a version with this change.  I tested on VoiceOver + Safari and NVDA + Firefox.  Let me know what you think of this version.

http://www.colorado.edu/physics/phet/dev/html/john-travoltage/1.3.0-dev.34/john-travoltage_en.html

**Comment by terracoda** [comment]
@jessegreenberg, I think it sounds great! And punctuation and capitalization looks good, too. 

The alert contains where the hand is rather than where the discharge started. I don't think this is a show stopper, but I thought the alert would contain the position of that hand where the discharged started.

@emily-phet, shall we go with where it ends for now? @jhung, any concerns?

**Comment by jessegreenberg** [comment]
Ah, good point @terracoda thanks.  The above commit should fix this, going to test locally before reassigning for review.

**Comment by jessegreenberg** [comment]
#237 and #235 make this sim very difficult to test on VO, the alerts and descriptions just aren't working well in VO and Safari.

**Comment by terracoda** [comment]
@jessegreenberg version 34 has the start of the discharge in the alert. Sounds good to me :-)

**Comment by jessegreenberg** [comment]
Great, thanks @terracoda! The latest version has the alert updated so that the it says where the hand was when the discharge started (as you suggested in https://github.com/phetsims/john-travoltage/issues/236#issuecomment-298097291).

Would you like to review a version with that change or can this be closed?

**Comment by terracoda** [comment]
@jessegreenberg, sorry I wasn't clear. Yes, the alert sounds good. I tested it. Closing the issue.

**Comment by jessegreenberg** [comment]
Ok sounds good, but version dev.34 does not have this change.

**Comment by terracoda** [comment]
@jessegreenberg, oh, it seemed like version dev.34 did have it. I guess I did not listen closely enough. I trust you. I can't do any more testing for another 8 or 9 hours.
